modules/core/src/main/java/io/fluo/webindex/core/DataUtil.java

```diff
@@ -22,6 +22,9 @@
 
 public class DataUtil {
 
+  public static String URL_SEP_REGEX = "[\\/\\?\\#]";
```

Comment:
I don't remember why I thought the escapes were necessary.  Removed in e577e45ff